v2/lints/cabf_br/lint_ext_tor_service_descriptor_hash_invalid_test.go

```diff
@@ -16,7 +16,7 @@ func TestTorDescHashInvalid(t *testing.T) {
 	}{
 		{
 			Name:           "Onion subject, no service descriptor extension, before util.CABV201Date",
-			InputFilename:  "dnsNameOnionTLD.pem",
+			InputFilename:  "onionSANEVBefore201.pem",
 			ExpectedResult: lint.NE,
 		},
 		{
```

Comment:
Note: `onionSANMissingServDescHash.pem` is updated to be an EV certificate, which does mean that coverage for the optionality (for DV) certs is missed. I was torn on whether to add a DV cert (or, more aptly, restore the existing cert as a DV test and add a new cert as an EV test). I wasn't sure to what degree we wanted to cover this with testing.

Reply:
I'm comfortable with the coverage you have in branch. Thanks for calling this out :+1: